src/proc/rotation-filter.h

```diff
@@ -0,0 +1,41 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "../include/librealsense2/hpp/rs_frame.hpp"
```

Comment:
Can this be replaced by something like
```
#include <librealsense2/hpp/rs_frame.hpp>
```
Same for the next line

Reply:
Done